20181216_VJAI_MLMeetup_#13.md

```diff
@@ -0,0 +1,49 @@
+# VN Machine Learning Meetup #13
+Form đăng ký tham dự: https://goo.gl/forms/pjMCFs7gDayq7sW93
+
+Chào cả nhà, đến với buổi Meetup cuối cùng của năm 2018, chúng ta sẽ đến với 2 bài toán nổi trội trong lĩnh vực xử lý ngôn ngữ tự nhiên là dịch máy (Machine Translattion) và đọc hiểu văn bản (Reading Comprehension). Hãy cùng đến xem xem máy tính đang tiến sát với con người trong việc xử lý ngôn ngữ đến đâu nhé.
+
+* Thời gian: 14h~18h Chủ nhật - 16/12/2018 (mở cửa đón khách 13h40 ~ 13h55)
+* Địa điểm: BizReach, Inc. 渋谷363清水ビル<br>
+〒150-0002, 3 Chome-6-3 Shibuya, Tokyo
+https://www.google.com/maps/place/%E6%B8%8B%E8%B0%B7363%E6%B8%85%E6%B0%B4%E3%83%93%E3%83%AB/@35.6579054,139.7029824,17z/data=!4m12!1m6!3m5!1s0x60188b596a766fc5:0xec3765750fe79ab1!2z5riL6LC3MzYz5riF5rC044OT44Or!8m2!3d35.6579054!4d139.7051711!3m4!1s0x60188b596a766fc5:0xec3765750fe79ab1!8m2!3d35.6579054!4d139.7051711
+
+Hạn cuối đăng ký là **23:59 thứ 3 - 11/12/2018**.
+
+https://goo.gl/forms/pjMCFs7gDayq7sW93
+
+Ai không kịp đăng ký trước thời hạn trên sẽ không được vào toà nhà nên ngoài ấn Going thì các bạn **nhớ điền vào form** trên nhé.
+
+Lần này ban tổ chức đã sắp xếp phòng rộng hơn nhưng cũng chỉ chứa được 50 người nên nếu số người đăng ký nhiều hơn, ban tổ chức sẽ áp dụng hình thức Lottery (như đã thông báo trên facebook group). Thêm nữa là việc vào toà nhà không dễ dàng nên các bạn tính toán thời gian cẩn thận để đến đúng giờ nhé. (TBC)
+
+Hẹn gặp lại cả nhà vào buổi sắp tới!
+
+Nội dung chương trình:
+---
+### Phạm Quang Khang
+AI Solutions Architect
+
+### Pay more attention to Attention mechanism
```

Comment:
quy ước viết title là viết hoa các từ quan trọng, kiểu như `Pay More Attention to Attention Mechanism`